projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -108,6 +108,47 @@ <h3 *ngIf="content.title" class="cx-card-title">
             >{{ action.name }}</a
           >
         </div>
+        <div *cxFeatureLevel="'4.2'" [ngSwitch]="action.event">
+          <button
+            *ngSwitchCase="'delete'"
+            class="cx-card-link card-link btn-link cx-action-link"
```

Comment:
We have an opportunity to drop any bootstrap classes we use here. Please consult @developpeurweb about how to style these buttons.

Reply:
For the new buttons let's use `<button class="link cx-action-link">` same as we do for the "Remove" action-links in the cart page.